OpenRA.Mods.Common/Traits/SupportPowers/AirstrikePower.cs

```diff
@@ -142,47 +162,50 @@ public void SendAirstrike(Actor self, WPos target, bool randomize = true, int at
 
 			self.World.AddFrameEndTask(w =>
 			{
-				PlayLaunchSounds();
-
-				Actor distanceTestActor = null;
-				for (var i = -info.SquadSize / 2; i <= info.SquadSize / 2; i++)
+				WPos? startPos = null;
+				foreach (var squadMember in info.Squad)
 				{
-					// Even-sized squads skip the lead plane
-					if (i == 0 && (info.SquadSize & 1) == 0)
-						continue;
+					var altitude = self.World.Map.Rules.Actors[squadMember.UnitType].TraitInfo<AircraftInfo>().CruiseAltitude.Length;
+					var attackRotation = WRot.FromFacing(attackFacing);
+					var delta = new WVec(0, -1024, 0).Rotate(attackRotation);
+					var targetPos = target + new WVec(0, 0, altitude);
+					var startEdge = targetPos - (self.World.Map.DistanceToEdge(targetPos, -delta) + info.Cordon).Length * delta / 1024;
+					var finishEdge = targetPos + (self.World.Map.DistanceToEdge(targetPos, delta) + info.Cordon).Length * delta / 1024;
+
+					startPos = startEdge;
 
-					// Includes the 90 degree rotation between body and world coordinates
-					var so = info.SquadOffset;
-					var spawnOffset = new WVec(i * so.Y, -Math.Abs(i) * so.X, 0).Rotate(attackRotation);
-					var targetOffset = new WVec(i * so.Y, 0, 0).Rotate(attackRotation);
+					PlayLaunchSounds();
 
-					var a = w.CreateActor(info.UnitType, new TypeDictionary
+					var spawnOffset = squadMember.SpawnOffset.Rotate(attackRotation);
```

Comment:
Please keep the comment
`// Includes the 90 degree rotation between body and world coordinates`
and the hardcoded 90 degree rotation here - it makes much more sense for these to be defined in body coordinates than world.

Reply:
Added back.